### PR TITLE
minor wording changes

### PR DESCRIPTION
**Pull request by mightyfrog**
as title says

### CODE REVIEW

File: lib/l10n/app_en.arb [hunk 1]
```diff
@@ -1,15 +1,14 @@
 {
     "home": "Home",
-    "event": "Event",
+    "event": "Events",
```
Comment (mightyfrog) [inline lib/l10n/app_en.arb:3]: I suggest "Past Events" though.

Comment (jiyuujin) [reply lib/l10n/app_en.arb]: @mightyfrog Updated to `Past Events`

**Review by jiyuujin — APPROVED** [review]
LGTM

**Review by kikuchy — APPROVED** [review]
LGTM 🙏

**Pull request merged**